tests/helm/testdata/goldenfile/terraform/collector_fields.output.yaml

```diff
@@ -753,6 +647,202 @@ data:
     export SUMOLOGIC_ACCESSID=
 
     bash /etc/terraform/custom.sh
+  sources.tf.json: |
+    {
+      "locals": {
+        "source_configs": {
+          "apiserver_metrics_source": {
+            "config-name": "endpoint-metrics-apiserver",
+            "name": "apiserver-metrics"
+          },
+          "control_plane_metrics_source": {
+            "config-name": "endpoint-control_plane_metrics_source",
+            "name": "control-plane-metrics"
+          },
+          "controller_metrics_source": {
+            "config-name": "endpoint-metrics-kube-controller-manager",
+            "name": "kube-controller-manager-metrics"
+          },
+          "default_events_source": {
+            "config-name": "endpoint-events",
+            "name": "events",
+            "properties": {
+              "default_date_formats": [
+                {
+                  "format": "epoch",
+                  "locator": "\\\"timestamp\\\":(\\\\d+)"
+                }
+              ]
+            }
+          },
+          "default_logs_source": {
+            "config-name": "endpoint-logs",
+            "name": "logs",
+            "properties": {
+              "default_date_formats": [
+                {
+                  "format": "epoch",
+                  "locator": "\\\"timestamp\\\":(\\\\d+)"
+                }
+              ]
+            }
+          },
+          "default_metrics_source": {
+            "config-name": "endpoint-metrics",
+            "name": "(default-metrics)"
```

Comment:
just curious how come default metrics has parenthesis around it? 🤔

Reply:
Someone long ago decided it should be this way, changing it is painful, and it doesn't really hurt anyone.